packages/tests/esm/joist-config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "codegenPlugins": [],
+  "codegenPlugins": ["joist-graphql-codegen"],
```

Comment:
It might make sense to do this in the main `integration/` test, too. Right now, we just copy the graphql-codegen.ts file as an example.